api/agent/iofs.go

```diff
@@ -3,6 +3,7 @@ package agent
 import (
 	"context"
 	"fmt"
+	"github.com/fnproject/fn/api/agent/drivers/docker"
```

Comment:
could you run goimports on this file please? https://godoc.org/golang.org/x/tools/cmd/goimports (it's easy to set up to auto-run on save/check-in/etc in your environment)

Reply:
yeah, this is necessary because we have to change the owner of the iofs directory to uid 1000 (so the function container can write to it), the change of ownership will fail if we run tests locally (not as root).

we could add make target to run the `make test` in docker as root, that container would need to have dind and golang, afaik.

anyway, to make sure, I will try PR without the sudo, so we can see where is fails...

Reply:
hmmm, gotcha. concerned about the local experience running fn too. I understand we want to have similar behavior b/w service and oss defaults here, I'm wondering what our options are; by default, map a host user pid we have perms on onto fn pid for the container (can allow configuring this or something)? something else? I'm not as familiar here

Reply:
omg, I put my previous comment to wrong place :-)) sorry